tower_eval/models/inference_handler.py

```diff
@@ -72,6 +74,27 @@ def assess_progress(
 
         return input_lines, processed_lines, metadata, num_processed_lines, total_lines
 
+    def apply_chat_template(self, input_line: str) -> str:
+        return input_line
+
+    def preprocess_lines(self, input_lines: List[str]) -> List[str]:
+        """ """
+        if self.strip:
+            input_lines = [input_line.strip() for input_line in input_lines]
+        else:
+            input_lines = [input_line for input_line in input_lines]
+        if self.use_chat_template:
+            if self.model_name() == "vllm":
+                logger.warning("Applying chat template to loaded instructions.")
+            else:
+                logger.warning(
```

Comment:
If this is a hard requirement, shouldn't we fail here and force the user to set it to False for the non-vllm models?
I think the warning might not be enough and can easily be ignored specially if we are running several models with the same config file.

Reply:
good point, will change this